ntfyme/utils/telegram/telegram.py

```diff
@@ -0,0 +1,22 @@
+
+import requests
+import toml
+import os
+USER_NAME = "https://t.me/notify_cli_bot"
+TOKEN = "7306967185:AAHnM_GHZwAAF8XGInfejGk00VqWtEfL4kQ"
```

Comment:
Please dont add your info. Make it <TOKEN> for now, and put a comment on how someone can add this token